python/ray/tests/test_output.py

```diff
@@ -337,15 +335,15 @@ def __repr__(self):
         out_str = proc.stdout.read().decode("ascii")
     else:
         out_str = proc.stderr.read().decode("ascii")
-    print(out_str)
```

Comment:
extra print

Reply:
Should I add it back or keep it removed. I believe it's a debug print that was left behind.